src/global.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "vscode-variables" {
+  export default function variables(
+    string: string,
+    recursive?: boolean,
+  ): string;
```

Comment:
do we need to export these types? is this needed by vscode ?

Reply:
This type definition is mainly for solving type checking issues.